titles/getting-started-rhdh/title-getting-started.adoc

```diff
@@ -7,31 +7,15 @@ include::artifacts/attributes.adoc[]
 
 As a developer, you can use {product} to experience a streamlined development environment. {product} is driven by a centralized software catalog, providing efficiency to your microservices and infrastructure. It enables your product team to deliver quality code without any compromises.
 
-//customer support links
-include::artifacts/snip-customer-support-info.adoc[]
-
 // rhdh overview
 include::modules/installation/con-rhdh-overview.adoc[leveloffset=+1]
 
 // Sizing
 include::modules/getting-started/ref-rhdh-sizing.adoc[leveloffset=+1]
 
-// supported configs and customization
-include::modules/getting-started/ref-rhdh-supported-configs.adoc[leveloffset=+1]
-include::assemblies/assembly-add-custom-app-file-openshift.adoc[leveloffset=+2]
 
 include::assemblies/assembly-bulk-importing-from-github.adoc[leveloffset=+1]
 
-include::assemblies/assembly-customizing-the-rhdh-homepage.adoc[leveloffset=+1]
-include::assemblies/assembly-customizing-the-quick-access-card-in-rhdh.adoc[leveloffset=+1]
-include::modules/getting-started/proc-customize-rhdh-tech-radar-page.adoc[leveloffset=+1]
-include::modules/getting-started/proc-customize-rhdh-learning-paths.adoc[leveloffset=+1]
-
-include::assemblies/assembly-customize-rhdh-theme.adoc[leveloffset=+1]
-
 include::modules/getting-started/con-servicenow-custom-actions.adoc[leveloffset=+1]
```

Comment:
Similar to bulk import This is just another plugin in RHDH, I think that the better placement would ve in the "Configuring dynamic plugins" section

Reply:
done